docs/4_4_1_can.adoc

```diff
@@ -199,11 +199,8 @@ For specification the boolean values `TRUE` and `FALSE` (see <<table-boolean-val
 | Data Length | 2 byte | Specifies the length of the Data argument in bytes.
 | Data | n byte | Stores the given frame data to transfer, whereby the valid length of the data depends on the CAN FD Format.
 h|Behavior
-3+|The CAN FD Transmit operation shall be provided by Network FMUs to initiate the transmission of a CAN FD frame.
-In case of directly connected Network FMUs (see <<common-concepts-direct-communication>>), the FMU importer forwards the operation directly to the receiving Network FMUs.
-If a Bus Simulation is involved (see <<common-concepts-composition-with-dedicated-bus-simulation-fmu>> and <<common-concepts-importer-with-integrated-bus-simulation>>), the FMU importer forwards the operation initially to the Bus Simulation, where the operation is distributed with respect to the simulated bus behavior.
-Depending on the simulation details, the Bus Simulation might response with a <<low-cut-can-confirm-operation, Confirm>>, <<low-cut-can-arbitration-lost-operation, Arbitration Lost>>, <<low-cut-can-bus-error-operation, Bus Error>> or <<low-cut-can-format-error-operation, Format Error>> operation.
-Depending on the <<low-cut-can-status-operation, status>> of the specified Network FMU further restrictions for CAN FD Transmit operations <<table-can-status-values, exist>>.
+3+|See <<table-can-transmit-operation, CAN Transmit>> behavior.
+The same circumstances also apply here.
```

Comment:
Maybe just "The behavior is identical to the <<table-can-transmit-operation, CAN Transmit>> behavior"?

Reply:
done